Sources/Commands/SwiftTestTool.swift

```diff
@@ -225,6 +233,12 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 print(test.specifier)
             }
 
+        case .codeCovPath:
+            let graph = try loadPackageGraph()
+            let buildParameters = try self.buildParameters()
+            let buildPlan = try BuildPlan(buildParameters: buildParameters, graph: graph, diagnostics: diagnostics)
```

Comment:
You can avoid a bunch of these costly operations since you just need the root package name. Something like:

```swift
let root = try getActiveWorkspace(). getWorkspaceRoot()
let rootManifest = workspace.loadRootManifests(packages: root.packages, diagnostics: diagnostics)[0]
print(codeCovAsJSONPath(buildParameters: buildParameters, packageName: rootManifest.name))
```